docs/polaris/queueing-and-running-jobs/example-job-scripts.md

```diff
@@ -24,7 +24,7 @@ cd ${PBS_O_WORKDIR}
 
 # MPI and OpenMP settings
 NNODES=`wc -l < $PBS_NODEFILE`
-NRANKS_PER_NODE=16
+NRANKS_PER_NODE=32
```

Comment:
I don't feel this example good for utilizing hardware threads.
When NRANKS_PER_NODE increases from 16 to 32.
rank 0 still uses logical core  0 and 1.
rank 16 uses logical core  32 and 33.
But the preferred way is rank 0 using logical core 0 and 32 to keep both threads on the same core.

So I feel the old example is only suitable for NRANKS_PER_NODE=16 and NRANKS_PER_NODE=32 needs a separate example.

Reply:
You're right, it would have to be `NDEPTH=1`, `NTHREADS=2` and `--cpu-bind=core`, in a second example.